src/index.css

```diff
@@ -11,3 +11,145 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* ----- Main styling ----- */
+body {
+  background-color: black;
+  color: white;
+}
+
+.header {
+  text-align: center;
+  font-size: 20px;
+}
+
+/* ----- Styling for album, cover and buttons -----*/
+.album-all {
+  margin: 10px;
+}
+
+.album-wrapper {
+  display: grid;
+  place-items: center;
+  grid-template-columns: 1fr;
+  grid-gap: 15px;
+}
+
+/* ----- Class for image for cover and buttons ----- */
+.cover-image {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  align-items: center;
+  width: 100%;
+  max-width: 500px;
+}
+
+/* Big image for cover */
+.cover-image img {
+  width: 100%;
+  height: auto;
+  display: block;
+  transition: opacity 0.3s ease;
+}
+
+.cover-image:hover img {
+  opacity: 0.7;
+  /* Reduce opacity when hovering */
+}
+
+/* Buttons and hovering of play-button*/
+.icon-buttons {
+  position: absolute;
+  opacity: 0;
+  transition: opacity 0.3s ease;
+}
+
+.cover-image:hover .icon-buttons {
+  opacity: 1;
+}
+
+.heart-button {
+  width: 50px;
+  height: 50px;
+  background-color: transparent;
+  border: none;
+}
+
+.play-button {
+  width: 50px;
+  height: 50px;
+  background-color: transparent;
+  border: none;
+}
+
+.play-button:hover {
+  transform: scale(1.2)
+}
+
+.dots-button {
+  width: 50px;
+  height: 50px;
+  background-color: transparent;
+  border: none;
+}
+
+/* Makes the icons white */
+svg {
+  fill: white;
+}
+
+/* ----- Album title and artist names ----- */
+.album-title {
+  font-size: 14px;
+  font-family: "Helvetica";
+}
```

Comment:
You could choose to put this code only in the "a" that you style under this. Same goes for .artist-name :)